### DISCUSSION TITLE
/add-certificate responds 403

### DISCUSSION THREAD

**Discussion by KuniyoshiKamimura** [shibayan/appservice-acmebot#262] [Q&A] [answered]
**Describe the bug**

**To Reproduce**
1. Access https://***.azurewebsites.net/add-certificate with chrome.
2. Gets `403`.

**Environment (please complete the following information):**
- App Service OS: Windows
- App Service Type: ASP.NET MVC
- Runtime Stack: .NET v4.8
- Certificate Type: naked

**Additional context**
App Service Authentication is enabled as document says.

**Comment by shibayan** [comment, marked as answer]
The App Service Authentication settings are not correct, please read the README and review the settings.

**Comment by KuniyoshiKamimura** [comment]
This is my Authentication settings.
![image](https://user-images.githubusercontent.com/73522066/103182066-858c2200-48eb-11eb-91cc-7997d7157778.png)
![image](https://user-images.githubusercontent.com/73522066/103182071-9046b700-48eb-11eb-8edd-198a1b740a33.png)

Anything wrong?

**Comment by KuniyoshiKamimura** [comment]
I noticed that if I enable App Service Authentication I can't access to app.
![image](https://user-images.githubusercontent.com/73522066/103469198-0db76f00-4da5-11eb-8f10-e911b72cba79.png)

I don't know what this setting means.

**Comment by KuniyoshiKamimura** [comment]
I was trying to install certification via "Azure Let's Encrypt" extension and it also failed.
After I struggled for many times, I realized the reason.
I've applied IP address restriction for App Service and it prevented Let's Encrypt to access to my site.
I didn't noticed for long time because I can access to the site.

I haven't confirmed that the failure in this topic have the same reason but maybe it is.
It's OK for me to close the case now.
Thank you for your cooperation.
